src/core/lib/common/util.ts

```diff
@@ -1028,3 +1028,10 @@ export function errorToJSON(error?: InternError): InternError | undefined {
     ...(showDiff ? { actual, expected } : {})
   };
 }
+
+export const originalTimers = {
```

Comment:
How about we export these from a new `lib/common/time` module, with a comment at the top explaining why we need it? Also, rather than dateNow, how about just `now`? Then we could do

```
import { setTimeout, now } from './common/time';
```

Reply:
That is much nicer and changes have been made